lib/users/get.js

```diff
@@ -0,0 +1,19 @@
+const async = require("async");
+const _ = require("lodash");
+const crypto = require("crypto");
+
+module.exports = function(req, res) {
+    req.pgdb.query("SELECT * FROM users", (error, result) => {
+        if(error) {
+            req.log.error({ error }, "Error while getting users");
+            return res.sendStatus(500);
+        }
+        let users = result.rows.map(row => {
```

Comment:
What? You really shouldn't use `global` at all, and you can't declare those to be contants, since they're scoped in an object. 

I assume you mean module/file-scope. But, why not use `const` whereever possible? It will make it harder to make certain programming errors.